pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -154,6 +154,13 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, pa *pav1alpha1.PodAutosc
 	if err != nil {
 		return fmt.Errorf("error checking pods for revision %s: %w", pa.Labels[serving.RevisionLabelKey], err)
 	}
+	// Manually set the desired scale to 0 if initial scale is 0, because otherwise
```

Comment:
Should this maybe live inside `scaler.go`?  I can see the SKS check is kind of dependent on this, but it feels more natural there.

Reply:
yes we can move this to scaler.go